apps/kyberswap-interface/src/pages/Earns/PositionDetail/PositionHistory.tsx

```diff
@@ -0,0 +1,87 @@
+
+const PositionHistory = ({ position }: { position: ParsedPosition }) => {
+  const theme = useTheme()
+  const { account } = useActiveWeb3React()
+
+  const { data: historyData } = usePositionHistoryQuery({
+    chainId: position.chainId,
+    tokenAddress: position.tokenAddress,
+    tokenId: position.id,
+    userAddress: account,
+  })
+
+  const createdTime = useMemo(() => {
+    const data = new Date(position.createdTime * 1000)
+    const hours = formatDateTime(data.getHours())
+    const minutes = formatDateTime(data.getMinutes())
+    const seconds = formatDateTime(data.getSeconds())
+    return `${hours}:${minutes}:${seconds} ${data.toLocaleDateString()}`
+  }, [position.createdTime])
+
+  const txHash = useMemo(() => {
+    if (!historyData) return ''
+    return [...historyData].reverse().find(item => item.type === PositionHistoryType.DEPOSIT)?.txHash || ''
+  }, [historyData])
+
+  return (
+    <InfoSection>
+      <Flex alignItems={'center'} justifyContent={'space-between'} marginBottom={2}>
+        <Text fontSize={14} color={theme.subText}>
+          {t`Created Time`}
+        </Text>
+        <Text>{createdTime}</Text>
+      </Flex>
+      {/* <Flex alignItems={'center'} justifyContent={'space-between'} marginBottom={2}>
+          <Text fontSize={14} color={theme.subText}>
+            {t`Liquidity Source`}
+          </Text>
+          <Flex alignItems={'center'} sx={{ gap: '6px' }}>
+            <Text fontSize={14} color={theme.subText}>{t`Value`}</Text>
+            <Text>$12,600</Text>
+          </Flex>
+        </Flex> */}
+      <Flex flexDirection={'column'} alignItems={'flex-end'} sx={{ gap: 2 }} marginBottom={2}>
+        {/* <Flex alignItems={'center'} sx={{ gap: '6px' }}>
+            <Text>345 KNC</Text> <Text>+ 12.65 JUP</Text> <Text>+ 0.18 ETH</Text>
+          </Flex> */}
+        {txHash ? (
+          <Flex alignItems={'center'} sx={{ gap: '6px' }}>
+            <Text fontSize={14} color={theme.subText}>{t`Tnx Hash`}</Text>
+            <Text
+              color={theme.blue2}
+              onClick={() => window.open(NETWORKS_INFO[position.chainId as ChainId].etherscanUrl + '/tx/' + txHash)}
+              sx={{ cursor: 'pointer' }}
+              marginRight={-1}
+            >{`${txHash.substring(0, 6)}...${txHash.substring(62)}`}</Text>
+            <CopyHelper color={theme.blue2} size={16} toCopy={txHash} />
+          </Flex>
+        ) : null}
+      </Flex>
+      {/* <Flex alignItems={'flex-start'} justifyContent={'space-between'}>
+          <Text fontSize={14} color={theme.subText}>{t`Past Actions`}</Text>
+          <Flex flexDirection={'column'} alignItems={'flex-end'} sx={{ gap: '6px' }}>
+            <Text>{t`Increased Liquidity`}</Text>
+            <Flex alignItems={'center'} sx={{ gap: '6px' }}>
+              <Text fontSize={14} color={theme.subText}>{t`Value`}</Text>
+              <Text>$12,600</Text>
+            </Flex>
+          </Flex>
+        </Flex> */}
```

Comment:
remove

Reply:
This is the UI that we will have when the position history is ready on the backend side, I have done this UI but temporarily hidden the un-ready parts